composer.json

```diff
@@ -38,7 +38,7 @@
         }
     ],
     "require": {
-        "php": ">=5.4",
+        "php": "~5.5 || ~7.0",
```

Comment:
How can you claim version 7 is supported if you're not testing it? There should be a symmetric change in `.travis.yml`.